projects/framework-nuke/extensions/tool-configs/nuke-script-publisher.yaml

```
@@ -38,6 +38,27 @@ engine:
           - exporter
         plugin: nuke_script_exporter
 
+  # Export the reviewable component.
+  - type: group
+    tags:
+      - component
+    options:
+      component: reviewable
+    #optional: False by default so is no need
+    #enabled: True by default so is no need
+    plugins:
+      - type: plugin
+        plugin: store_component
+      - type: plugin
+        tags:
+          - collector
+        plugin: nuke_node_collector
+        ui: nuke_node_selector
```

Should we add this one as well? In case the user pass the node_name from the tool-config, we check if it exists. (Like we do with the cameras for PS and Maya)

```suggestion
        ui: nuke_node_selector
    - type: plugin
        tags:
          - validator
        plugin: nuke_node_exists_validator
        ui: validator_label
```